.ci_support/linux_64_.yaml

```diff
@@ -1,7 +1,7 @@
 cdt_name:
 - cos6
 channel_sources:
-- conda-forge,defaults
+- conda-forge
```

Comment:
@xylar Hey, I found that `default` was taken out. Is it okay?

Reply:
Yes, that's expected.  Conda-forge is making this change as part of updating (rerendering) all feedstocks